### DISCUSSION TITLE
Is Tremor compatible with Tailwind v4.0?

### DISCUSSION THREAD

**Discussion by nicholasmanske** [tremorlabs/tremor-npm#1010] [Q&A]
Curious if/how Tremor can be configured using Tailwind v4.0 (which does not have a tailwind.config.js file): https://tailwindcss.com/blog/tailwindcss-v4-alpha

Thanks!

**Comment by tiarebalbi** [comment]
I don't think so, and since the adoption of Radix in the `raw` version, I decided to move away from `tremor`. Tremor started by having well-defined components that were very flexible to customize, with a centralized theme.

But this new philosophy of `copy-and-paste` will create continuous refactoring among the imported components, especially in the case of a breaking change from Radix. (ignoring the fact of a new set of dependencies that needs to be added)
